site/docs/contribute/build-test.md

```diff
@@ -62,14 +62,22 @@ export DOCKER_NAMESPACE=m4d-system
 # build a local kind cluser
 make kind
 
-# deploy the the cluster 3rd party such as cert-manager and vault
-make cluster-prepare
+# deploy vault
+make -C charts vault
+make -C charts wait-for-vault
 
-# build all docker images and push them to the local registry
-make docker
+# check if cert-manager is already installed
+kubectl get namespaces
```

Comment:
Consider removing. `make kind` creates a new cluster and this is a manual process anyway (if it's already installed then the next command will say so)

Reply:
@roee88 The quick start documentation instructs to check if the cert-manager is already installed (although it doesn't say how to do so) and says to only run the command to do the install if it isn't.  Should it be changed there as well?

Reply:
No. These run against a kind cluster that was created for testing. There is no sane scenario where cert-manager is installed there and not through the helm install that comes next